docs/doc/90-contributing/08-how-to-write-scalar-functions.md

```diff
@@ -23,16 +23,12 @@ Scalar functions (sometimes referred to as User-Defined Functions / UDFs) return
 └─────┘                    └──────┘
 ```
 
-
 ### Knowledge before writing the eval function
 
 #### Logical datatypes and physical datatypes.
 
 Logical datatypes are the datatypes that we use in Databend, and physical datatypes are the datatypes that we use in the execution/compute engine.
```

Comment:
We use logical datatypes in Databend and physical datatypes in the execution/compute engine.